libcodechecker/libhandlers/store.py

```diff
@@ -342,9 +341,7 @@ def get_analysis_statistics(inputs, limits):
                     LOG.debug("Copying file '%s' to analyzer statistics "
                               "ZIP...", compilation_db)
                     statistics_files.append(compilation_db)
-            elif inp_f in ['compiler_includes.json',
-                           'compiler_target.json',
-                           'compiler_info.json',
+            elif inp_f in ['compiler_info.json',
```

Comment:
Maybe we should keep the files. If an older version was used to analyze the project, but a newer to store those files will be left out from the zip. @martong @bruntib can we remove those files from the zip?

Reply:
We should have either the new `compiler_info.json` or the old pair of includes and target json in the failed zip. In both cases we can reproduce the failure. If the info file is missing and there are only an include and target file then put those into the zip.

Reply:
Created a small helper function for this.